streams/tr.m3u

```diff
@@ -76,8 +76,6 @@ http://stream.taksimbilisim.com:1935/ciftcitv/smil:ciftcitv.smil/playlist.m3u8
 https://live.artidijitalmedya.com/artidijital_ciftcitv/ciftcitv/playlist.m3u8
 #EXTINF:-1 tvg-id="Cine5.tr",Cine5 (720p)
 https://cdn-cine5tv.yayin.com.tr/cine5tv/cine5tv/playlist.m3u8
-#EXTINF:-1 tvg-id="CNNTurk.tr",CNN Türk (720p)
-https://live.duhnet.tv/S2/HLS_LIVE/cnnturknp/playlist.m3u8
```

Comment:
Works.

Reply:
Yes, it works. But the problem is that it is a violation of rights.

Reply:
The following address has been closed to public access. I kindly ask you to check again.

https://live.duhnet.tv/S2/HLS_LIVE/cnnturknp/playlist.m3u8